ChatSecure/Classes/OTRAppDelegate.m

```diff
@@ -569,7 +569,9 @@ - (void) userNotificationCenter:(UNUserNotificationCenter *)center willPresentNo
 
 - (void) userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)())completionHandler {
     NSDictionary *userInfo = response.notification.request.content.userInfo;
-    if (userInfo[kOTRNotificationThreadKey] == nil) {
+    if ([kOTRNotificationTypeNone isEqualToString:userInfo[kOTRNotificationType]]) {
+        // Nothing
+    } else if ([kOTRNotificationTypeSubscriptionRequest isEqualToString:userInfo[kOTRNotificationType]]) {
```

Comment:
@N-Pex I think this should be done in reverse because userInfo[kOTRNotificationType] can be nil.

```obj-c
NSString *type = userInfo[kOTRNotificationType];
if (![type isKindOfClass:[NSString class]]) { return; } // handles nil or wrong class
if ([type isEqualToString:.....]) etc
```